src/Control/Coroutine.purs

```diff
@@ -87,19 +88,27 @@ instance monadRecCo :: (Functor f, Monad m) => MonadRec (Co f m) where
         Left s1 -> go s1
         Right a -> return a
 
+-- | Lift a command from the functor `f` into a one-step `Co`routine.
+liftCo :: forall f m a. (Functor f, Monad m) => f a -> Co f m a
+liftCo fa = Co \_ -> return (Right (map pure fa))
+
 -- | Change the underlying `Monad` for a `Co`routine.
 hoistCo :: forall f m n a. (Functor f, Functor n) => (forall a. m a -> n a) -> Co f m a -> Co f n a
 hoistCo n (Co m) = Co \_ -> map (map (hoistCo n)) <$> n (m unit)
 hoistCo n (Bind e) = runExists (\(Bound a f) -> bound (hoistCo n <<< a) (hoistCo n <<< f)) e
 
+-- | Change the functor `f` for a `Co`routine.
+interpret :: forall f g m a. (Functor f, Functor m) => (forall a. f a -> g a) -> Co f m a -> Co g m a
+interpret n (Bind e) = runExists (\(Bound a f) -> bound (interpret n <<< a) (interpret n <<< f)) e
```

Comment:
I think it'd be nice if this and `hoistCo` read the same way, since they are very similar in structure. In fact, maybe they could both use the same helper function which took a natural transformation _and_ a monad morphism (`mapCo`?)